### PR TITLE
Added the advanced section, for selecting the tab completion

### PR DESCRIPTION
**Pull request by SachinS10-lab**
Added toggle button for advanced section.

Image

--------------------

For Simple Configuration

Image

---------------------

For Advanced Configuration

Image

---------------------

I also add an information message below the box for better user experience. Is this Okay?

Image

### CODE REVIEW

**Comment by fbricon** [comment]
Please reread the instructions in #137 
> the advanced section, where the tab completion can be selected?

**Comment by SachinS10-lab** [comment]
> Please reread the instructions in #137
> 
> > the advanced section, where the tab completion can be selected?

Okay, I understand. I simplified the code to become only one combobox for both selections. Will change that.

**Comment by SachinS10-lab** [comment]
Added the tab completion combo in the advanced section.

image

-----

Simple section.

image

File: webviews/src/App.tsx [hunk 1]
```diff
@@ -254,50 +254,45 @@ function App() {
...
+            <ModelList
+              className="model-list"
+              label="Chat model"
+              value={chatModel}
+              onChange={(e) => setChatModel(e?.value ?? null)}
+              status={getModelStatus(chatModel)}
+              options={modelOptions}
+              progress={chatModel ? modelPullProgress[chatModel] : undefined}
+              disabled={!enabled}
+              tooltip="This model will be used for Chat and Tab Completion"
```
Comment (fbricon) [inline webviews/src/App.tsx:281]: This model will be used for Chat

**Comment by fbricon** [comment]
See comments in screenshot

Screenshot 2024-11-18 at 14 35 35

**Comment by fbricon** [comment]
also, the 1st information message should have a single *.
The new one can have 2 *

**Comment by fbricon** [comment]
FTR, the build is still broken

**Comment by SachinS10-lab** [comment]
Hi @fbricon, 
In App.test.tsx, the tabModelId is accepting only null value. Due to that Build is failing. I'm working on it.

**Comment by SachinS10-lab** [comment]
Hi @fbricon, 
I'm adding a new PR for this fix. I accidently pushed package.json files in this PR.

New PR: https://github.com/redhat-developer/vscode-paver/pull/149

**Pull request closed without merging**